spark/src/main/scala/geotrellis/spark/filter/Implicits.scala

```diff
@@ -16,9 +16,8 @@ trait Implicits {
       extends TileLayerRDDFilterMethods[K, V, M]
 
   implicit class withSpaceTimeToSpatialMethods[
-    K: SpatialComponent: TemporalComponent,
+    K: SpatialComponent: TemporalComponent: λ[α => M[α] => Functor[M, α]]: λ[α => Component[M[α], Bounds[α]]],
```

Comment:
It's definitely space voodoo the first time you see it, yeah. For technical reasons they can't be bounds on M, so I'll explain that.